test/proxy.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * The talkback proxy for e2e tests. When making a request to the API during testing,
+ * it tries to use the responses it previously saved in `/test/tapes` folder. If no
+ * response is found there, it makes an actual request, and saves the response for
+ * future use.
+ * This makes it possible for the e2e tests to run without internet, and makes the
+ * tests less flaky due to changes in the API or API data.
+ */
+const talkback = require('talkback')
+const host = 'https://api.openverse.engineering/v1'
+
+const tapeNameGenerator = (tapeNumber) => `response-${tapeNumber}`
+
+const opts = {
+  host,
+  port: 3000,
+  path: './test/tapes',
+  record: talkback.Options.RecordMode.NEW,
```

Comment:
FWIW this appears to be the default value for `record` so we can leave it out if we keep it as `NEW`.

However, I'm wondering if we should actually change this to `DISABLED` and set the fallback mode to `NOT_FOUND`. I'm leaning in this direction because ideally we'd prevent _any_ external request from hitting the actual API and get some noticeable errors happening if we've forgotten to add tapes for new routes, or if our matching rules aren't working as expected.

To make it easy to update tapes, it'd be convenient to look at `process.argv` for a `--update-tapes` param then only use `RecordMode.NEW` if that's present.